Structure_Engine/Convert/PanelToFEMesh.cs

```diff
@@ -0,0 +1,108 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+
+using BH.oM.Structure.Elements;
+using BH.oM.Geometry;
+using BH.Engine.Geometry;
+using BH.oM.Reflection.Attributes;
+using System.Collections.Generic;
+using System.Linq;
+using System.ComponentModel;
+using BH.Engine.Analytical;
+using BH.Engine.Structure;
+
+namespace BH.Engine.Structure
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+        [Description("Converts a Panel with three or four control points to a femesh with a single Face. This is not a method to discretise a Panel, it simply converts a simple Panel to an identical feMesh.")]
+        [Input("panel", "Panel to be converted to a feMesh.")]
+        [Output("feMesh", "feMesh converted from a Panel.")]
+
+        public static FEMesh PanelToFEMesh(this Panel panel)
+        {
+            List<Point> points = new List<Point>();
+            List<Edge> edges = panel.ExternalEdges;
+            List<Face> faces = new List<Face>();
+            if (BH.Engine.Structure.Compute.NullCheck(panel) == false)
+            {
+                return null;
+            }
```

Comment:
Need to make null checks before accessing any property of the object. Also, small coding style tweak

```suggestion
            if (!BH.Engine.Structure.Compute.NullCheck(panel))
            {
                return null;
            }
            List<Point> points = new List<Point>();
            List<Edge> edges = panel.ExternalEdges;
            List<Face> faces = new List<Face>();

```